language/defined_spec.rb

```diff
@@ -48,6 +48,16 @@
 end
 
 describe "The defined? keyword when called with a method name" do
+  it "does not execute the method" do
+    defined?(DefinedSpecs.side_effects).should == "method"
+    ScratchPad.recorded.should != :defined_specs_side_effects
```

Comment:
This could be brittle since the ScratchPad is not cleared before.
Could you add this?
```ruby
  before :each do
    ScratchPad.clear
  end
```

Reply:
Done. I also changed 'execute the method' to 'call the method.'